MedvedevKS/lab_4/lab_4.cpp

```diff
@@ -0,0 +1,105 @@
+// lab_4.cpp: ���������� ����� ����� ��� ����������� ����������.
+//
+
+#include "stdafx.h"
+#include <conio.h>
+#include <iostream>
+using namespace std;
+
+double intPower(double number, int n)
+{
+	double result = 1;
+	for (int i = 0; i < abs(n); i++)
+	{
+		result *= number;
+	}
+	if (n < 0)
+	{
+		result = 1 / result;
+	}
+	return result;
+}
+
+int factorial(int number)
+{
+	if (number < 0)
+	{
+		return 0;
+	}
+	if (number > 12)
+	{
+		return -1;
+	}
+	int result = 1;
+	for (int i = 2; i <= number; i++)
+	{
+		result *= i;
+	}
+	return result;
+}
+
+double SquareRoot(double number)
+{
+	double result = 0;
```

Comment:
Well, looks like it's Nobel Prize candidat since square root of -2 is equal to 0.6
![image](https://user-images.githubusercontent.com/19354025/48379679-0ecd4400-e6e6-11e8-8858-67f75ba24bc5.png)